libs/sdk-backend-spi/src/workspace/dashboards/layout/facade/interfaces.ts

```diff
@@ -7,21 +7,37 @@ import {
     IFluidLayoutSize,
     IFluidLayoutSizeByScreen,
     ResponsiveScreenType,
-} from "./fluidLayout";
+} from "../fluidLayout";
 
 /**
  * TODO: RAIL-2869 add docs
  * @alpha
  */
 export interface IFluidLayoutColumnMethods<TContent> {
     raw(): IFluidLayoutColumn<TContent>;
+
+    testRaw(pred: (column: IFluidLayoutColumn<TContent>) => boolean): boolean;
+    test(pred: (column: IFluidLayoutColumnMethods<TContent>) => boolean): boolean;
+
     index(): number;
+    indexIs(index: number): boolean;
+
     size(): IFluidLayoutSizeByScreen;
     sizeForScreen(screen: ResponsiveScreenType): IFluidLayoutSize | undefined;
-    style(): IFluidLayoutColumn<TContent>["style"];
+    hasSizeForScreen(screen: ResponsiveScreenType): boolean;
+
     content(): TContent | undefined;
-    row(): IFluidLayoutRowMethods<TContent>;
+    hasContent(): boolean;
+    contentEquals(content: TContent): boolean;
+    contentIs(content: TContent): boolean;
+
+    isFirstInRow(): boolean;
     isLastInRow(): boolean;
+
+    row(): IFluidLayoutRowMethods<TContent>;
+
+    // TODO: expose this?
```

Comment:
Is this actionable now?

Reply:
We don't use this "style" property anywhere and I'm not sure about it's purpose (+ now we have theming and customization possibilities). Should I remove it? (probably yes?)